examples/CMakeLists.txt

```diff
@@ -30,6 +30,16 @@ blt_add_executable(
   DEPENDS_ON ${example_depends})
 list(APPEND umpire_examples allocator)
 
+blt_add_executable(
+  NAME backtrace_example
+  SOURCES backtrace_example.cpp
+  DEPENDS_ON ${example_depends})
+list(APPEND umpire_examples backtrace_example)
+
+blt_add_target_link_flags(
```

Comment:
This needs guarding for MSVC?

Reply:
Adding `rdynamic` on Windows issues a warning about unknown flag, so it should be guarded. See here: https://dev.azure.com/davidbeckingsale/Umpire/_build/results?buildId=2769&view=results